src/main/java/at/hannibal2/skyhanni/config/core/config/Position.kt

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright (C) 2022 NotEnoughUpdates contributors
+ *
+ * This file is part of NotEnoughUpdates.
+ *
+ * This file was translated to Kotlin and modified, 2024.
+ *
+ * NotEnoughUpdates is free software: you can redistribute it
+ * and/or modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation, either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * NotEnoughUpdates is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with NotEnoughUpdates. If not, see <https://www.gnu.org/licenses/>.
+ */
+package at.hannibal2.skyhanni.config.core.config
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.config.ConfigGuiManager.getEditorInstance
+import at.hannibal2.skyhanni.test.command.ErrorManager
+import com.google.gson.annotations.Expose
+import io.github.notenoughupdates.moulconfig.annotations.ConfigLink
+import io.github.notenoughupdates.moulconfig.gui.GuiScreenElementWrapper
+import net.minecraft.client.Minecraft
+import net.minecraft.client.gui.ScaledResolution
+import java.lang.reflect.Field
+
+class Position @JvmOverloads constructor(
+    x: Int,
+    y: Int,
```

Comment:
```suggestion
    x: Int = 0,
    y: Int = 0,
```

in the java one these were also defaulted to 0